Add documentation on processing spatial transcriptomics libraries (#47)
AlexsLemonade/scpca-docs: docs/processing_information.md

Any cell barcodes that appeared only in CITE-seq data were discarded.
Cell barcodes that were present only in the RNA-seq data (i.e., did _not_ appear in the CITE-seq data) were assigned zero counts for all ADTs. 
When cells were [filtered based on RNA-seq content](#filtering-cells) after quantification, the CITE-seq count matrix was filtered to match.

## Spatial Transcriptomics

**_Data Available Soon_**

### Mapping and quantification using Space Ranger

Processing spatial transcriptomics libraries requires two steps - gene expression quantification and tissue detection. 

> Currently, Alevin-fry is not capable of tissue detection, but only reports gene expression. 
> Therefore, we utilized [10X Genomics' Space Ranger](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger) to obtain both gene expression and spatial information. 

Reviewer: Merging these two sentences, in the hopes of a bit less redundancy. I also want to sort of imply that AF shouldn't really be expected to do tissue id. (What I would really want is a `spaceranger imageprocess` command, but I'm not gonna get that.)
```suggestion
In the absence of independent tissue detection methods to use with Alevin-fry, we used [10X Genomics' Space Ranger](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger) to obtain both gene expression and spatial information. 
```

> [`spaceranger count`](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/using/count) takes a microscopic slide image and FASTQ files as input and performs alignment, tissue detection, and spot barcode and UMI counting. 

Reviewer: I simplified the end of this sentence, and flipped the order of the input files to match the "expression, image" order that we have throughout this section. 
```suggestion
[`spaceranger count`](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/using/count) takes FASTQ files and a microscopic slide image as input and performs alignment, quantification, and tissue detection for each spot. 
```

> In contrast to Alevin-fry which maps reads to a [reference transcriptome index](#reference-transcriptome-index), Space Ranger aligns transcript reads to the reference genome using STAR ([Dobin _et al_., 2012](https://doi.org/10.1093/bioinformatics/bts635)).

Reviewer: ```suggestion
In contrast to Alevin-fry, which maps reads to a [reference transcriptome index](#reference-transcriptome-index), Space Ranger aligns transcript reads to the reference genome using STAR ([Dobin _et al_., 2012](https://doi.org/10.1093/bioinformatics/bts635)).
```
